src/mongocxx/test/database.cpp

```diff
@@ -267,7 +266,7 @@ TEST_CASE("Database integration tests", "[database]") {
     client mongo_client{uri{}};
     bsoncxx::stdx::string_view database_name{"database"};
     database database = mongo_client[database_name];
-    bsoncxx::stdx::string_view collection_name{"collection"};
+    std::string collection_name{"collection"};
```

Comment:
Similarly, I think with the comparators, you could just make this a straightforward const char*.